package/kedro_viz/server.py

```diff
@@ -511,10 +498,32 @@ def _call_viz(
 
         data = format_pipeline_data(pipeline, catalog)
 
+    enhanced_data = None
+    if KEDRO_VERSION.match(">=0.16.0"):
+        enhanced_data = context._hook_manager\
```

Comment:
Also, if there are multiple registrations of this hook implementation, hook call will [return a list of results](https://pluggy.readthedocs.io/en/latest/index.html#collecting-results), which, I believe, will break the frontend.